unicodetools/org/unicode/text/UCD/IdentifierInfo.java

```diff
@@ -60,14 +58,14 @@
     private static IdentifierInfo info;
 
     static IdentifierInfo getIdentifierInfo() {
-	try {
-	    if (info == null) {
-		info = new IdentifierInfo();
-	    }
-	    return info;
-	} catch (final Exception e) {
-	    throw (RuntimeException) new IllegalArgumentException("Unable to access data").initCause(e);
-	}
+        try {
```

Comment:
FYI  I untabified this file when I thought I needed to make changes here. Later I found that I could just add to the removals.txt file.